### PR TITLE
Merge feature/members only

### PR DESCRIPTION
**Pull request by sebGilR**
The members-only project created with the following features:
- Users can be created, edited, and deleted.
- Posts can be created only by logged in users.
- Post author names can only be seen by logged in users.

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  </thead>
+
+  <tbody>
+    <% @posts.each do |post| %>
+      <tr>
+        <td><%= post.title %></td>
+        <td><%= post.body %></td>
+        <% if user_signed_in? %>
+          <td><%= post.user.name %></td>
+        <% end %>
```
Comment (euqueme) [inline app/views/posts/index.html.erb]: ###### `index`

* [x] the author is displayed only if a user is signed in (uses ApplicationController helper method) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_index_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication))

* [x] View-related logic can be encapsulated in View helpers (app/helpers/\*_helper.rb)

* [x] There should be absolutely NO logic in views.

  > Views are only for displaying information that became available to it from its corresponding controller not generating it. **Resources**
* [Where Do I Put My Code?](http://codefol.io/posts/Where-Do-I-Put-My-Code)
* [The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers)
* [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @sebGilR and @enelesmai :balloon: :lollipop: 

### You've done a great job with this project, your code looks impeccable, :sparkles:  claps to you for using build instead of new for creating new posts, you should feel very proud of your work :clap: :clap: :clap: :1st_place_medal:  I would like you to do minor changes and add a little more to your Readme file so please keep reading

#### General

- [x] Please add a product screenshot, don't use the login page it could be your root page

- [x] Add a Usage section describing with examples and screenshots on how to create and signup a User, login, and create a Post. (a walkthrough video would be nice to have in this section too)

- [ ] Include an Automated Test section in your Readme file if you used Minitest or Rspec include the command to run your test cases.

#### Project Specific

- [x] I've put changes in your code below but basically let's not put logic directly in the views let's use helper methods for that :point_down: 

### That's it, team! :raised_hands: 

### Please :pray: do the changes :man_technologist: :woman_technologist: and submit back for a code review when you are done :memo: 

Happy Coding! :rocket: 

Any question? please ask on slack **@ Maru** :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :vertical_traffic_light:

**Comment by enelesmai** [comment]
> > ### Pull Request under the review by Maria Eugenia Queme P. (TSE)
> 
> ### Hello @sebGilR and @enelesmai 🎈 🍭
> ### You've done a great job with this project, your code looks impeccable, ✨ claps to you for using build instead of new for creating new posts, you should feel very proud of your work 👏 👏 👏 🥇 I would like you to do minor changes and add a little more to your Readme file so please keep reading
> #### General
> * [x]  Please add a product screenshot, don't use the login page it could be your root page
> * [x]  Add a Usage section describing with examples and screenshots on how to create and signup a User, login, and create a Post. (a walkthrough video would be nice to have in this section too)
> * [ ]  Include an Automated Test section in your Readme file if you used Minitest or Rspec include the command to run your test cases.
> 
> #### Project Specific
> * [x]  I've put changes in your code below but basically let's not put logic directly in the views let's use helper methods for that 👇
> 
> ### That's it, team! 🙌
> ### Please 🙏 do the changes 👨‍💻 👩‍💻 and submit back for a code review when you are done 📝
> Happy Coding! 🚀
> 
> Any question? please ask on slack **@ Maru** 👩‍💻
> 
> PROJECT STATUS: REQUESTED CHANGES 🚦

The changes suggested for adding screenshots and user instructions in the readme file and moving logic to helpers were implemented, for this specific project automated tests were not included as per the requirements.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,79 @@
...
+
+
+## Screenshot
+
+![image](https://user-images.githubusercontent.com/5160907/81008694-f771a180-8e18-11ea-901a-c2ec4be49735.png)
+
+
+## Usage
+
+This is the main page, you will be able to create posts until you are logged in.
```
Comment (mosesogwo) [inline README.md]: This should read, _This is the main page, you will **not** be able to create posts until you are logged in._

**Review by mosesogwo — APPROVED** [review]
**Hi team:clap: You have done well:+1: Project is approved.**

You can now merge this pull request, but before you do that, see my comment on your README.

**Cheers:stars:**

### Status: Approved :heavy_check_mark:

File: README.md (hunk 2 above)
Comment (sebGilR) [reply README.md]: Fixed, thank you

**Pull request merged**